addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -309,25 +309,26 @@ function makeActionManager(env) {
         const options = { clearBreadcrumbs: true };
         let actionRequest = null;
         if (state.action) {
+            const context = {};
+            if (state.active_id) {
+                context.active_id = state.active_id;
+            }
+            if (state.active_ids) {
+                context.active_ids = parseActiveIds(state.active_ids);
+            } else if (state.active_id) {
+                context.active_ids = [state.active_id];
+            }
             // ClientAction
             if (actionRegistry.contains(state.action)) {
                 actionRequest = {
+                    context,
                     params: state,
                     tag: state.action,
                     type: "ir.actions.client",
                 };
             } else {
                 // The action to load isn't the current one => executes it
                 actionRequest = state.action;
-                const context = { params: state };
```

Comment:
removing it from here is out of the scope of this PR, so feel free to keep it here. My point was not to introduce it somewhere else.

Reply:
Woops sorry, I reverted this change to keep this part intact. As I said some parts of the code rely on it so it's indeed better to keep it :)